Makefile

```diff
@@ -36,6 +36,12 @@ distclean:
 	$(RM) -r $(BUILDDIR) $(TARGETS)
 	$(ECHO) " done"
 
+BUSTED_TESTS:=$(wildcard spec/**/*.lua)
```

Comment:
I don't see this recursive use of wildcards in https://www.gnu.org/software/make/manual/html_node/Wildcards.html
There is no `**`, I fear.